src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -339,7 +339,7 @@ public function present($cart, $shouldSeparateGifts = false)
         if ($total_discount) {
             $subtotals['discounts'] = array(
                 'type' => 'discount',
-                'label' => $this->translator->trans('Discount', array(), 'Shop.Theme.Checkout'),
+                'label' => $this->translator->trans('Total discounts', array(), 'Shop.Theme.Checkout'),
```

Comment:
'label' => $this->translator->trans('Discount(s)', array(), 'Shop.Theme.Checkout'),

Reply:
I changed it to "Discount(s)"